dashboard/src/components/Header/ContextSelector.test.tsx

```diff
@@ -153,3 +155,18 @@ it("shows an error creating a namespace", () => {
   // The error will be within the modal
   expect(wrapper.find(CdsModal).find(Alert)).toExist();
 });
+
+it("disables the create button if not allowed", () => {
+  const clusters = {
+    currentCluster: "foo",
+    clusters: {
+      foo: {
+        currentNamespace: "default",
+        namespaces: ["default"],
+        canCreateNS: false,
+      },
+    },
+  } as IClustersState;
+  const wrapper = mountWrapper(getStore({ clusters }), <ContextSelector />);
+  expect(wrapper.find(".flat-btn")).toBeDisabled();
```

Comment:
A few lines above we're doing `.find(CdsButton).filterWhere(b => b.text() === "Create Namespace");`, it'd be worth updating that to use the new class name (so it's clearly the same node in both cases and sets the right precedent for future changes).